devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -571,5 +599,54 @@ public enum DisplayResolution
             0x00, // lower columns address =0
             0x10, // upper columns address =0
         };
+		
+		/// <summary>
+        /// Reset display controller.
+        /// </summary>        
+        private void Reset()
+        {            
+            GpioPin rstPin = _gpioController.OpenPin(_resetPin, PinMode.Output);
+            rstPin.Write(PinValue.High);
+            Thread.Sleep(1);                // VDD goes high at start, pause for 1 ms            
+            rstPin.Write(PinValue.Low);     // Bring reset low
+            Thread.Sleep(10);               // Wait 10 ms
+            rstPin.Write(PinValue.High);    // Bring out of reset
+            Thread.Sleep(1);           
+        }
+
+        /// <summary>
+        /// Cleanup resources.
+        /// </summary>
+        /// <param name="disposing">Should dispose managed resources.</param>
+        protected virtual void Dispose(bool disposing)
+        {
+            if (_disposed)
+            {
+                return;
+            }
+
+            if (disposing)
+            {
+                _gpioController?.ClosePin(_resetPin);
+                if (_shouldDispose)
+                {                                   
+                    _gpioController?.Dispose();
+                    _gpioController = null;
+                }
+				
+				_i2cDevice?.Dispose();
```

Comment:
alignment issue that can be fixed easilly with ctrl + k + d.
And also please add StyleCop, those things should be catch with it.